doc/classes/NavigationRegion3D.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="NavigationRegion3D" inherits="Node3D" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A region of the navigation map.
+		A traversable 3D region that [NavigationAgent2D]s can use for pathfinding.
```

Comment:
A region is not necessarily traversable, it can be disabled, locked or obstructed. The navigation mesh the region holds defines what is traversable while the navigation layers define what is accessible for a specific agent.

Reply:
AFAIK this a redundant detail, so I didn't change it (I fixed the mention of 2D though).

Basically, a room is still an area you can traverse, even if:
- The door is locked and you don't have the key (locked)
- The door is broken (obstructed)
- It's only for staff (different layer)

And not being traversable when disabled doesn't need to be documented, unless only specific parts of it can be disabled.

I don't know much about navigation though, so correct me if there's an important distinction.

Reply:
I am ok with you don't want to change it here and keep the description short. There have been a few incidents were users had trouble to understand how to change what is traversable (by changing navigation mesh) and what is accessible (by changing navigation layers) but that information might be added and amplified later at another place.